### DISCUSSION TITLE
MsalClientException NO_CURRENT_ACCOUNT

### DISCUSSION THREAD

**Discussion by ekahannah** [AzureAD/microsoft-authentication-library-for-android#1846] [General]
Hi everyone, 
I recently found out that users are being logged out of my app when they used the same login credentials in a different app. This usually happens when the token is refreshing. Any ideas how to solve this issue? 

This is similar to this issue:
"User has logged in the azure account in Android system account manager and he uses the same account to login in the app. I don't understand how the token is refreshing. I don't get it how could I handle the login properly. I assume that some of users don't add azure account to android account manager."
https://github.com/AzureAD/microsoft-authentication-library-for-android/issues/1189

**Comment by negoe** [comment]
@ekahannah Please create an issue [here](https://github.com/AzureAD/microsoft-authentication-library-for-android/issues/new?assignees=&labels=&projects=&template=bug_report.md&title=).
